fennel.lua

```diff
@@ -1164,8 +1164,7 @@ local function compile1(ast, scope, parent, opts)
             end
             local compiled = compile1(newAST, scope, parent, opts)
             exprs = compiled
-        elseif(type(ast) == "table") then
```

Comment:
What's the purpose of removing this check? Isn't the rest of this only sensible on a table?

Reply:
Since the only way for this condition to be true in this part of the code (see the comment about recursion after `macroexpand`) because of the parent `if isList(ast)`, this can safely go back to an `else` since we're still operating under the parent condition being true.